src/framework/ui/internal/uiconfiguration.h

```diff
@@ -118,6 +118,7 @@ class UiConfiguration : public IUiConfiguration, public async::Asyncable
 
     void updateSystemThemeListeningStatus();
     void synchThemeWithSystemIfNecessary();
+    void synchAccentColorWithSystemIfNecessary();
```

Comment:
Two little things I've found:

The shortened form of  `synchronize` is `sync` (not `synch`).

Reply:
I spelled it as `synch` to match the spelling of the existing `synchThemeWithSystemIfNecessary` method name

Reply:
@ryan-roche I'll leave this to a team member to decide. 🙂